### DISCUSSION TITLE
Trémulant

### DISCUSSION THREAD

**Discussion by Jean-Gabriel1982** [GrandOrgue/GoOdf#72] [General]
Hello i have 3 trémulants in my 3 keyboards, configured by default 
- period 160
- start rate 8
- stop rate 8
- amplitude modulation depth 18

I want tremulants more marked but when I change the values, it seems there is no change

**Comment by larspalo** [comment]
The period value will affect the oscillation rate (speed) and the amplitude modulation depth will change the amount. Those two values are most important when shaping the tremulant effect. The start and stop rate is of less importance - but feel free to experiment with them too.

Also, be sure to remember that the tremulants must be set correctly for the windchests (where the ranks/pipes) are placed. The adding of the tremulants to the manual is mainly for the divisional combination referencing...

**Comment by larspalo** [comment]
To be more specific: lower Period values means a faster rate and a higher Amplitude Modulation Depth values means more volume variation.

Just for comparing some of the value combinations I put together a small sample set as a demonstration. Not all combinations will be "good", but then again what people judge to be good varies a lot! Note that it's very much possible to have multiple tremulants for the same windchest and interesting things might happen when they are combined... 

[TestTremProperties.zip](https://github.com/larspalo/GOODF/files/13465252/TestTremProperties.zip)

**Reply by Jean-Gabriel1982** [reply]
Thank you very much fore those explanations !
I will check that and tell you 🙂
